pkg/chains/signing.go

```diff
@@ -222,11 +234,22 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 	// Now mark the TektonObject as signed
 	if err := MarkSigned(ctx, tektonObj, o.Pipelineclientset, extraAnnotations); err != nil {
 		return err
+	} else {
+		logger.Infof("Recording count for stored payload for %v", tektonObj)
+		measureMetrics("mrcount", o.Recorder, ctx)
 	}
-
 	return nil
 }
 
+func measureMetrics(metrictype string, mtr MetricsRecorder, ctx context.Context) {
+	logger := logging.FromContext(ctx)
+	logger.Infof("Type of Metrics Recorder ", mtr, metrictype)
```

Comment:
Let's remove this to avoid noise, or change it to `Trace` level. Also, every call to this function in this file is preceded by a logged message. Let's not duplicate these messages. Remove one.